### PR TITLE
Add: Bluefish.Bluefish version 2.2.12rc2

### LINKED ISSUE #19826: yumi-bios bluefish
PackageIdentifier: bluefish
PackageVersion: 2.2
PackageLocale: en-US
Publisher: bluefish
PackageName: bluefish
License: proprietary
ShortDescription: IDE for web development
Installers:

Architecture: x64
InstallerType: exe
InstallerUrl: https://www.bennewitz.com/bluefish/stable/binaries/win32/Bluefish-2.2.12rc2-setup.exe
ManifestType: singleton
ManifestVersion: 1.0.0

### PR DESCRIPTION
**Pull request by vedantmgoyal9**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] Have you validated your manifest locally with `winget validate --manifest <path>`? 
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.0 schema](https://github.com/microsoft/winget-cli/blob/master/doc/ManifestSpecv1.0.md)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----
ARP:
image
Fixes #19826 

###### Microsoft Reviewers: [Open in CodeFlow](https://portal.fabricbot.ms/api/codeflow?pullrequest=https://github.com/microsoft/winget-pkgs/pull/20443)

### CODE REVIEW

**Comment by ImJoakim** [comment]
-----------------------------------------------------
Manifest: manifests\b\Bluefish\Bluefish\2.2.12rc2
-----------------------------------------------------
URI: http://bluefish.openoffice.nl/index.html, Validation result: Passed
URI: https://bfwiki.tellefsen.net/index.php/Manual_2_ToC, Validation result: Passed
URI: http://bluefish.openoffice.nl/features.html, Validation result: Passed
URI: http://bluefish.svn.sourceforge.net/viewvc/bluefish/trunk/bluefish/COPYING, Validation result: Failed, Http status code: NotFound 
URI: https://www.bennewitz.com/bluefish/stable/binaries/win32/Bluefish-2.2.12rc2-setup.exe, Validation result: Passed

****************************************************************************************
Manifest Content Validation
****************************************************************************************
Status: Completed
##[error] Result: Failed

File: manifests/b/Bluefish/Bluefish/2.2.12rc2/Bluefish.Bluefish.locale.en-US.yaml [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+PackageLocale: en-US
+Publisher: The Bluefish Developers
+PublisherUrl: http://bluefish.openoffice.nl/index.html
+PublisherSupportUrl: https://bfwiki.tellefsen.net/index.php/Manual_2_ToC
+#PrivacyUrl: 
+#Author: 
+PackageName: Bluefish
+PackageUrl: http://bluefish.openoffice.nl/features.html
+License: GNU GPL
+LicenseUrl: http://bluefish.svn.sourceforge.net/viewvc/bluefish/trunk/bluefish/COPYING
```
Comment (ItzLevvie) [inline manifests/b/Bluefish/Bluefish/2.2.12rc2/Bluefish.Bluefish.locale.en-US.yaml:15]:
```suggestion
LicenseUrl: https://sourceforge.net/p/bluefish/code/HEAD/tree/trunk/bluefish/COPYING
```
Try this.

**Comment by ImJoakim** [comment]
You can find this information by doing the following:

1. Click on `Details` behind the check that failed

image

2. Click on `View more details on Azure Pipelines`

image

3. Click on the check that failed

image

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

File: manifests/b/Bluefish/Bluefish/2.2.12rc2/Bluefish.Bluefish.installer.yaml [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+MinimumOSVersion: 10.0.0.0
+InstallModes:
+- interactive
+- silent
+- silentWithProgress
+Installers:
+- InstallerLocale: en-US
+  Architecture: x86
+  InstallerType: nullsoft
+#  Scope:
```
Comment (ItzLevvie) [inline manifests/b/Bluefish/Bluefish/2.2.12rc2/Bluefish.Bluefish.installer.yaml:15]:
```suggestion
  Scope: machine
```

**Review by ItzLevvie — CHANGES_REQUESTED** [review]

**Review by ItzLevvie — APPROVED** [review]

**Pull request merged**